src/iterable.cr

```diff
@@ -75,4 +75,24 @@ module Iterable(T)
   def chunk_while(reuse : Bool | Array(T) = false, &block : T, T -> B) forall B
     each.chunk_while(reuse, &block)
   end
+
+  # Same as `each.accumulate`.
+  def each_accumulated
```

Comment:
Why add these when `each.accumulate` is fewer keystrokes?

Reply:
Because they're more efficient. It's not always just about the keystrokes.

Reply:
In this case they are exactly the same

Reply:
Yeah, we probably shouldn't add those.

Reply:
Oh, they're actual aliases, didn't look at the code itself 🤦‍♂️

Reply:
*Every* non-abstract method in `Iterable` is an alias, because that's all it does.

Reply:
To be honest, I wanted more or less the same API as Ruby but at this point I'm not sure it was a good idea